packages/tests/unit-tests/test-cases/expected/preact/jquery-api-wrapper.tsx

```diff
@@ -6,7 +6,7 @@ import { MyTypeReturn, MyType } from "./jquery-api";
 
 export default class Widget extends BaseComponent {
   getValue(arg: MyType): MyTypeReturn {
```

Comment:
No we can return `undefined`
Probably we should fix return type to reduce TS errors (even if this code will transpile in JS for jQuery)